### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by LaskoA**
Please check my solution (image file is pretty big so please advice how to fix it). Thank you.

### CODE REVIEW

**Comment by LaskoA** [comment]
Managed to decrease image size from 1GB to 400MB - please check it with next command:
"docker pull laskoa/docker-weather-api-slim"

Thank you.

**Comment by LaskoA** [comment]
Now image is "only" 341Mb, please use next command:
"docker pull laskoa/docker-weather-api-slim-v3"

Thanks in advance.

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+API_KEY=7c508243647f4c4b971122001220609
```
Comment (DmytroSvirsa) [inline .env:1]: that is your secret API_KEY, never push stuff like this,

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN apt-get update \
+    && apt-get -y install libpq-dev gcc \
+    && pip install psycopg2
+
+RUN pip install -r requirements.txt
+
+COPY . .
+
+CMD ["python", "manage.py", "runserver"]
```
Comment (DmytroSvirsa) [inline Dockerfile:18]: error python: can't open file '/app/manage.py': [Errno 2] No such file or directory

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+
+RUN apt-get update \
+    && apt-get -y install libpq-dev gcc \
+    && pip install psycopg2
```
Comment (DmytroSvirsa) [inline Dockerfile:12]: you can add it to requirements.txt

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]

**Comment by LaskoA** [comment]
Please check now after fixes, use command "docker pull laskoa/docker-weather-api" (old images were deleted from Docker).

P.S.: If you don`t mind I would leave next string in dockerfile (not in requirements.txt):
    RUN apt-get update \
    && apt-get -y install libpq-dev gcc \
    && pip install psycopg2

The thing is slim-image did not work without it so I would like to keep it seperately for the future as a reminder :)

Thank you.

**Review by DmytroSvirsa — APPROVED** [review]
Yep, well done!

**Pull request closed without merging**